astropath/bayesian.py

```diff
@@ -194,6 +196,64 @@ def px_Oi(box_hwidth, frb_coord, eellipse, cand_coords,
     else:
         return np.array(p_xOis)
 
+def px_Oi_healpix(transient, nside, cand_coords, theta_prior, step_size=0.1,
+                  coord_sys='C',
+                  debug = False):
+    """
+
+    Args:
+        transient (np.ndarray):
+            Healpix probability values
+        nside (int):
+            Healpix NSIDE
+        cand_coords (astropy.coordinates.SkyCoord):
+        theta_prior (dict):
```

Comment:
although it is obvious, it might still be useful to write some description of cand_coords and theta_prior, and their units.

Reply:
done